modules/cluster-logging-kibana-visualize.adoc

```diff
@@ -59,3 +59,12 @@ For example:
 ----
 $ oc exec elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6k -- es_load_kibana_ui_objects <user-name>
 ----
+
+[NOTE]
+====
+The metadata of the Kibana objects such as visualizations, dashboards, and so forth are stored in Elasticsearch with the *.kibana.{user_hash}* index format. You can obtain the *user_hash* using the `userhash=$(echo -n $username | sha1sum | awk '{print $1}')` command.
+By default, the Kibana *shared_ops* index mode enables all users with cluster admin roles to share the index, and this Kibana object metadata is saved to the *.kibana* index.
```

Comment:
@anpingli @mijicherian
Please take a look. I reworded the `shared_ops` paragraph to reflect Jeff Cantrill's request to not describe setting index mode

Reply:
LGTM